_kurzdokus/sd-2023/sd-Lindek-Kueper.md

```diff
@@ -0,0 +1,34 @@
+---
+title: ProjectSurvival4All - Praktische Tipps für die Projektarbeit im Studium und darüber hinaus
+people:
+    - Jonas Lindek
+    - Tom Küper
+teaserimage: Lindek-Kueper-Teaser.png
```

Comment:
@Rabytez Dieses Bild existiert nicht. Deshalb wird auf der Übersichtssseite eine leere Kachel angezeigt.
https://th-koeln.github.io/mi-bachelor-screendesign-projekte/
Bitte ein entsprechendes Bild hinzufügen und einen neuen Pull Request erstellen.

Reply:
Die Pfade in der Markdowndatei zu den anderen Bildern sind auch falsche. Bitte entsprechend korrigieren und mit in den Pull Request packen.